docs/nodle-wallets/nodle-cash/Hotspots.mdx

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 5
+---
+
+# Nodle Hotspots
+
+![Title](/img/docs/nodle-cash/HotSpots.png)
+
+### Do I need the main Nodle app to use the Hotspot app?
+
+Yes, the Hotspot app relies on the main Nodle app, which should be installed on your primary phone. Here's why: your primary device holds something called a "private key" for your Hotspot, which is like a password that gives it the right to collect your Hotspot earnings.
+
+The Hotspot app, on the other hand, doesn't have this private key. It only has a "public key," which is a sort of identity marker but doesn't give it the power to transfer your earnings. This means you can't collect your earnings through the Hotspot app alone—you'll need the main Nodle app for that.
```

Comment:
@nikitnikoks @bluewitch this seems a little incorrect from the technical point of view

Reply:
Removed error until I can get clarification

Reply:
The Hotspot app cannot be used without the Nodle app, because it is configured by the Nodle app. When the config is happening, both public and private keys are generated on the Nodle app and sent to the Hotspot app. The public key is required to earn rewards, while the private key is required to sign transactions, and, therefore, collect rewards.